cmd/ConfigLoader/main.go

```diff
@@ -173,6 +174,11 @@ func main() {
 				Value:   "Info",
 				Usage:   "Log level",
 			},
+			&cli.StringFlag{
+				Name:    "jenkinsjoburl",
+				Aliases: []string{"jj"},
+				Usage:   "VM renter jenkins job url",
```

Comment:
Does it fail when you don't include the parameter? 

We shouldn't fail the CLI cmd if the parameter and value aren't specified. I know Vlad has used a default "Value" for certain parameters, but I'm not sure if the reason for that is that the CLI framework required the parameters OR if the null value caused an issue.

Reply:
Per documentation:
String looks up the value of a local StringFlag, returns "" if not found.

I added default value "N/A" also.

Reply:
What's the decision here?

Reply:
The decision is to not provide a default value as "N/A". When the arg is not provided, goLang will default to "". And you can query db with that "" string.